pegasus/helpers/hoc_helpers.rb

```diff
@@ -23,6 +24,10 @@ def create_session_row_unless_unsampled(attrs)
   # Decide whether the session should be sampled.
   weight = DCDO.get('hoc_activity_sample_weight', 1).to_i
 
+  # Don't sample for cartoon network (or any company). We always need to create a
+  # session row in order to show the correct call to action on the congrats page.
+  weight = 1 unless attrs[:company].nil?
```

Comment:
I think Asher's suggestion is more readable, the unless clause is too easy to miss.